datafusion/common/src/dfschema.rs

```diff
@@ -819,11 +841,9 @@ mod tests {
         let left = DFSchema::try_from_qualified_schema("t1", &test_schema_1())?;
         let right = DFSchema::try_from_qualified_schema("t1", &test_schema_1())?;
         let join = left.join(&right);
-        assert!(join.is_err());
         assert_eq!(
-            "Schema error: Schema contains duplicate \
-        qualified field name \"t1\".\"c0\"",
-            &format!("{}", join.err().unwrap())
+            join.unwrap_err().to_string(),
```

Comment:
I cleaned these tests up to use a more standard `unwrap_err()` and `to_string()` pattern